src/actions.ts

```diff
@@ -89,7 +103,50 @@ export async function createJiraIssue(
   }
 
   // return the issue
-  return client.findIssue(newIssue.key);
+  return jiraClient.findIssue(newIssue.key);
+}
+
+type FullCreateIssueActionProperties = Omit<
+  CreateIssueActionProperties & { storedActionData: false },
+  'storedActionData'
+>;
+function isValidCreateIssueActionProperties(
+  actionProperties: any,
+): actionProperties is FullCreateIssueActionProperties {
+  let valid = !!actionProperties;
+  valid &&= '' < actionProperties.project;
+  valid &&= '' < actionProperties.summary;
+  valid &&= '' < actionProperties.classification;
+  return valid;
```

Comment:
This seems a bit arcane - I don't think i've ever seen someone use `&&=` in code before, nor `'' < string` for string existence. It seems like it works, but can we just do this for more standard code?


```suggestion
  const { project, summary, classification } = (actionProperties || {});
  return !!project && !!summary && !!classification;
```

Reply:
It is arcane, yes. It has the benefit of asserting not just existence but also string-ness. Maybe we don't need to be that defensive. I can update.

Reply:
See newest changeset for something hopefully much less arcane.

Reply:
Actually, I'm wrong on this behavior. I'll remove the `<`.